tests/cla_ext_test.sh

```diff
@@ -23,40 +23,61 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 PORT_NODE1=3000
 PORT_NODE2=3001
 PORT_NODE3=3002
+PORT_NODE4=3003
+PORT_NODE5=3004
 
+#refuse false
 OUT_NODE1=$(mktemp /tmp/node1.XXXXXX)
 #DB3="-W /tmp/node3 -D sled"
 #DB3="-W /tmp/node3 -D sneakers"
-$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE1 -C tcp:4224 -e incoming -r flooding -n node1 -s tcp://127.0.0.1:4225/node2 $STATUS_REPORTS 2>&1 &> $OUT_NODE1 &
+$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE1 -C tcp:port=4224 -e incoming -r flooding -n node1 -s tcp://127.0.0.1:4225/node2 $STATUS_REPORTS 2>&1 &> $OUT_NODE1 &
 PID_NODE1=$!
-echo node3 pid: $PID_NODE1
-echo node3 out: $OUT_NODE1
-echo node3 port: $PORT_NODE1
+echo node1 pid: $PID_NODE1
+echo node1 out: $OUT_NODE1
+echo node1 port: $PORT_NODE1
 
+#refuse true
 OUT_NODE2=$(mktemp /tmp/node2.XXXXXX)
 #DB3="-W /tmp/node3 -D sled"
 #DB3="-W /tmp/node3 -D sneakers"
-$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE2 -C tcp:4225:true -e incoming -r epidemic -n node2 -s tcp://127.0.0.1:4224/node1 -s tcp://127.0.0.1:4226/node3 $STATUS_REPORTS 2>&1 &> $OUT_NODE2 &
+$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE2 --ext tcp:refuse-existing-bundles=true -C tcp:port=4225 -e incoming -r flooding -n node2 -s tcp://127.0.0.1:4226/node3 $STATUS_REPORTS 2>&1 &> $OUT_NODE2 &
 PID_NODE2=$!
-echo node5 pid: $PID_NODE2
-echo node5 out: $OUT_NODE2
-echo node5 port: $PORT_NODE2
+echo node2 pid: $PID_NODE2
+echo node2 out: $OUT_NODE2
+echo node2 port: $PORT_NODE2
 
+#refuse true
 OUT_NODE3=$(mktemp /tmp/node3.XXXXXX)
-$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE3 -C tcp:4226 -e incoming -r epidemic -n node3 -s tcp://127.0.0.1:4226/node2 $DB4 $STATUS_REPORTS 2>&1 &> $OUT_NODE3 &
+$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE3 --ext tcp:refuse-existing-bundles=true -C tcp:port=4226 -e incoming -r flooding -n node3 -s tcp://127.0.0.1:4227/node4 $DB4 $STATUS_REPORTS 2>&1 &> $OUT_NODE3 &
 PID_NODE3=$!
-echo node4 pid: $PID_NODE3
-echo node4 out: $OUT_NODE3
-echo node4 port: $PORT_NODE3
+echo node3 pid: $PID_NODE3
+echo node3 out: $OUT_NODE3
+echo node3 port: $PORT_NODE3
+
+#refuse false
+OUT_NODE4=$(mktemp /tmp/node4.XXXXXX)
+$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE4 --ext tcp:refuse-existing-bundles=false -C tcp:port=4227 -e incoming -r flooding -n node4  -s tcp://127.0.0.1:4228/node5 $DB4 $STATUS_REPORTS 2>&1 &> $OUT_NODE4 &
+PID_NODE4=$!
+echo node4 pid: $PID_NODE4
+echo node4 out: $OUT_NODE4
+echo node4 port: $PORT_NODE4
+
+#refuse false
+OUT_NODE5=$(mktemp /tmp/node5.XXXXXX)
+$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE5 --ext tcp:refuse-existing-bundles=false -C tcp:port=4228 -e incoming -r flooding -n node5 $DB4 $STATUS_REPORTS 2>&1 &> $OUT_NODE5 &
```

Comment:
yes, it is absolutely possible to have it as local parameter. but instead of reflecting back to global options, it overrides the global option for this specific cla. if no global option is set, then the default value for that option is used (in this case the extension is turned off)

Reply:
okay, overriding is also fine but then the test should also cover setting the refuse config via `-C` to ensure that it keeps working :)